packages/js-drive/lib/identity/masternode/handleNewMasternodeFactory.js

```diff
@@ -79,13 +79,11 @@ function handleNewMasternodeFactory(
       );
 
       // Create a document in rewards data contract with percentage
-      result.push(
-        await createRewardShareDocument(
-          dataContract,
-          masternodeIdentifier,
-          operatorIdentifier,
-          proRegTxPayload.operatorReward,
-        ),
+      await createRewardShareDocument(
```

Comment:
Shouldn't we add this to the result if the returned value is not null?

Reply:
Why? It's not an identity.